app/models/better_together/user.rb

```diff
@@ -44,6 +43,13 @@ def person
       super.present? ? super : build_person
     end
 
+    # def person= arg
```

Comment:
@Kneeba921 please clean up this commented test code from our pair programming session experiment.